src/fsharp/SyntaxTree.fs

```diff
@@ -477,8 +477,9 @@ type SynExpr =
 
     | AnonRecd of
         isStruct: bool *
-        copyInfo:(SynExpr * BlockSeparator) option *
-        recordFields:(Ident * SynExpr) list *
+        copyInfo: (SynExpr * BlockSeparator) option *
```

Comment:
Could we have the range of the `with-` keyword in the tree as well?
This would be useful for Fantomas if the code has a comment after the `with-` for example.
```fsharp
type A = { X: int; Y: int }
let a = { X = 1; Y = 2 }
let b = {| a with- // foo 
                                  Y |}
```